### PR TITLE
Initial integration and refactoring into new repository

### PR DESCRIPTION
**Pull request by rranjan3**
Refactored docker files to move out of primary codebase directory.
Added .gitignore to home directory. Updated main.rs, engine.rs,
service.rs to bring them in sync with sawtooth-core master.

Signed-off-by: Rajeev Ranjan <rajeev2.ranjan@intel.com>

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -20,18 +20,18 @@ services:
   poet-2-engine:
     container_name: poet-2-engine
     build:
-      context: ../../..
-      dockerfile: ./consensus/poet2/core/poet-2-engine.dockerfile
+      context: ./
+      dockerfile: ./poet-2-engine.dockerfile
```
Comment (askmish) [inline docker-compose.yaml]: Name should be 'poet2'  instead of 'poet-2'

File: src/core/src/engine.rs [hunk 2]
```diff
@@ -140,14 +139,13 @@ impl Engine for Poet2Engine {
...
-                            DevmodeMessage::Ack => {
+                            ResponseMessage::Ack => {
                                 let block_id = BlockId::from(message.content);
                                 info!("Received ack message from {:?}: {:?}", sender_id, block_id);
                             }
                         },
 
-                        // Devmode doesn't care about peer notifications
-                        // or invalid blocks.
+						//Ignoring invalid blocks for now
```
Comment (askmish) [inline src/core/src/engine.rs]: Indentation of comment, should align with preceding code

File: unit_poet_2.yaml [hunk 3]
```diff
@@ -18,14 +18,16 @@ version: "2.1"
 services:
 
   unit-poet-2:
+    container_name: poet-2-test
     build:
-      context: ../../..
-      dockerfile: ./consensus/poet2/core/poet-2-tests.dockerfile
+      context: ./
+      dockerfile: ./poet-2-engine.dockerfile
```
Comment (askmish) [inline unit_poet_2.yaml]: 'poet-2' should be as 'poet2'

File: unit_poet_2.yaml [hunk 4]
```diff
@@ -18,14 +18,16 @@ version: "2.1"
 services:
 
   unit-poet-2:
+    container_name: poet-2-test
```
Comment (askmish) [inline unit_poet_2.yaml]: 'poet2' instead of 'poet-2'

**Comment by askmish** [comment]
Please rename all file to 'poet2' instead of using 'poet-2'

**Comment by rranjan3** [comment]
Addressed all the comments

**Pull request merged**